src/translations/nl_NL.translation

```diff
@@ -0,0 +1,787 @@
+        <name>labelVolunteersWizardCampaignInformation</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for Close Existing Campaign --></label>
+        <name>labelVolunteersWizardClone</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for Copy Volunteers checkbox on Volunteers Wizard --></label>
+        <name>labelVolunteersWizardCopyVolunteers</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- error message for not choosing an existing campaign or sample jobs --></label>
+        <name>labelVolunteersWizardErrorExistingOrSample</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- error message for not choosing Sample Jobs along with Sample Shifts --></label>
+        <name>labelVolunteersWizardErrorNoJobsWithShifts</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for first shift date on Volunteers Wizard --></label>
+        <name>labelVolunteersWizardFirstShiftDate</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- The help text displayed at the top of the Volunteers Wizard page. --></label>
+        <name>labelVolunteersWizardHelp</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- help text for the Volunteers Wizard Jobs &amp; Shifts section. --></label>
+        <name>labelVolunteersWizardJobAndShiftHelp</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Label for the Volunteers Wizard Volunteer Jobs &amp; Shifts section --></label>
+        <name>labelVolunteersWizardJobsAndShifts</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- The title for the main section of the Volunteers Wizard. --></label>
+        <name>labelVolunteersWizardNewCampaignTitle</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for new shift date on Volunteers Wizard --></label>
+        <name>labelVolunteersWizardNewShiftDate</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for Volunteer Hours Status picklist on Volunteers Wizard --></label>
+        <name>labelVolunteersWizardOverrideHoursStatus</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- name of sample jobs --></label>
+        <name>labelVolunteersWizardSampleJobName</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for number of sample jobs --></label>
+        <name>labelVolunteersWizardSampleJobs</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- name of sample shifts --></label>
+        <name>labelVolunteersWizardSampleShiftName</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- label for number of sample shifts --></label>
+        <name>labelVolunteersWizardSampleShifts</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- the title of the Volunteers Wizard page. --></label>
+        <name>labelVolunteersWizardTitle</name>
+    </customLabels>
+    <customLabels>
```

Comment:
it looks like they did not translate a single custom label!  95% of these should be translated.  @mpusto any idea why they would have skipped all these?  Doesn't seem worth integrating yet with these missing.